Fix RA 25mm and DepthCharge playing sounds

reaperrr opened the pull request:
25mm explosions and DepthCharge Report should be silent.

Fixes regressions from #12633.

reaperrr committed:
Fix RA 25mm and DepthCharge playing sounds

25mm explosions and DepthCharge Report should be silent.

pchote commented:
Did #12615 break our ability to override nodes with empty values?

reaperrr commented:
> Did #12615 break our ability to override nodes with empty values?

Nope, does not seem to be the case, I just tested this on bleed right before #12615 was merged and bleed from Dec 23rd, the issue persists (and yes, I made sure to recompile), so it looks like it was already broken much earlier.

SoScared commented:
Another regression found with the DepthCharge. I didn't notice at first, it's very subtle but in the release it has sort of "underwater-impact" sound coupled with the explosion sound when the DepthCharge hits its target. When missing its target (submarines on the move) the water-impact sound play as intended. With direct hits the DepthCharge cancels out the water-impact sounds and only plays the explosion sound.

25mm sound works as intended.

reaperrr committed:
Fix DepthCharge hit explosion

The splash anim + h2obomb2.aud should always play, regardless of whether a submarine was hit.

reaperrr commented:
Updated.
